playground/README.md

```diff
@@ -0,0 +1,68 @@
+# SignIn Widget Playground
+
+## The playground App
+In a nutshell, the playground application demonstrates customer hosted sign-in widget case, which loads sign-in widget assets and bootstrap widget via configurations.
+The configurations have been done in two folds
+
+- [widgetrc](https://github.com/okta/okta-signin-widget#the-widgetrc-config-file)
+- `./main.js`
+
+What `main.js` does is initialize sign-in widget using settings from `widgetrc` plus simple success handler.
+
+## Mock server
+
+While in `widgetrc` you can point `baseUrl` to any Okta tenants, it can also be set to mock server, which is started at `http://localhost:3000` when starting playground application.
```

Comment:
While starting the playground application, the `baseUrl` in `widgetrc` can be used to point to any Okta tenant. Similarly it can also be set to point to the mock server, which runs at `http://localhost:3000`